server/players.py

```diff
@@ -76,6 +79,39 @@ def __init__(
         self._game = lambda: None
         self._game_connection = lambda: None
 
+    @property
```

Comment:
Don't use `@property`, it doesn't give you anything, only hiding call functions + add wrappers, without proper IDE impossible to detect with naked eye.
Use property only if you're trying to not to break same API during refactoring.
Better to use `set_` methods. It's more explicit.

Reply:
It has no side effects beyond setting the backing value. Sounds like a perfectly fine usage of @property to me.

Reply:
In fact it has. If you see below, you'll find condition that decides what to do with Rating instance. It is hiding behavior.

Reply:
I was talking about *side effects*. The only thing setting this value ever affects is the value returned by the getter. This is a valid use case for a property, what you're describing restricts `@property` to trivial cases.